plugins/pom-analyzer/src/main/java/eu/f4sten/pomanalyzer/PomAnalyzerInjectorConfig.java

```diff
@@ -51,9 +48,4 @@ public DatabaseUtils bindDatabaseUtils(PostgresConnector pc, JsonUtils json, Ver
         var dslContext = DSL.using(c, SQLDialect.POSTGRES);
         return new DatabaseUtils(dslContext, json, version);
     }
-
-    @ProvidesIntoSet
-    public Module bindJacksonModule() {
```

Comment:
Might be a stupid question! Why is this removed?

Reply:
The module was empty.